src/cryptoadvance/specterext/devhelp/service.py

```diff
@@ -26,6 +27,8 @@ class DevhelpService(Service):
             "app": app,
             "flask": flask,
             "flask_login": flask_login,
+            "current_user": current_user,
+            "namespace": console.namespace,
```

Comment:
Can we remove this again?
![image](https://user-images.githubusercontent.com/60378539/186642439-fbec498a-4bbe-48d4-abe3-d0258b52b3a1.png)

Reply:
done